src/sentry/static/sentry/app/views/organizationEventsV2/table.jsx

```diff
@@ -47,8 +48,8 @@ class Table extends React.Component {
       return <LoadingIndicator />;
     }
 
-    return events.map(event => (
-      <PanelItem key={event.id} className={getGridStyle(fields.length)}>
+    return events.map((event, idx) => (
+      <PanelItem key={idx} className={getGridStyle(fields.length)}>
```

Comment:
are events not unique in the table? when would we have duplicated events?

Reply:
We won't have duplicated events but a row might now represent a single event In the errors and csp tab each row is a grouping of events and hence has no event id associated